package.json

```diff
@@ -30,25 +30,20 @@
     "postversion": "npm install --package-lock-only --ignore-scripts --silent",
     "tsx": "tsx",
     "test": "node ./scripts/test.mjs",
-    "lint": "eslint '{src,tests,scripts,benches}/**/*.{js,mjs,ts,mts,jsx,tsx}'",
-    "lintfix": "eslint '{src,tests,scripts,benches}/**/*.{js,mjs,ts,mts,jsx,tsx}' --fix",
-    "lint-shell": "find ./src ./tests ./scripts -type f -regextype posix-extended -regex '.*\\.(sh)' -exec shellcheck {} +",
+    "lint": "matrixai-lint",
+    "lintfix": "matrixai-lint --fix",
```

Comment:
We still want to be explicit about the globbing pattern options.

Reply:
This depends on https://github.com/MatrixAI/js-lint/pull/16 to bring those options in.